site/layouts/_default/_markup/render-image.html

```diff
@@ -0,0 +1,8 @@
+{{- $originalSrc := .Destination | safeURL -}}
+{{- $localImgPath := path.Join "/site/static/docs" site.Params.docs_version $originalSrc -}}
+{{- /* This shouldn't be needed but we have a weird folder structure with version included... */ -}}
+{{- $src := urls.JoinPath "/docs" site.Params.docs_version $originalSrc -}}
+{{- $config := imageConfig $localImgPath -}}
+{{- $classes := "d-block img-fluid" -}}
```

Comment:
Have you found any use cases needing to add this `.d-block` class? Doesn't seem to be useful in the current usage within the guides.

Reply:
We are using `d-block` in blog and I'd swear at some point we needed `d-block` for some browser to work correctly, but maybe it was IE and SVG files.

Reply:
OK, no problem. I don't mind keeping it for consistency or safety if you think it can be useful.

Reply:
I don't mind dropping it, I just can't recall the reason we were using it. I *think* it was IE, but maybe @ffoodd or @mdo remembers.

Reply:
My guess it allowed to drop bottom margin that came with inline and inline-block images in some cases, especially with `img-fluid`.

Could also relate to how images behave as flex children.

Not quite sure though 🤷 Would need some history digging.

Reply:
I decided to keep it for now, since it's easier to center an image if needed later.